scripts/check-pkg-for-release.js

```diff
@@ -126,14 +133,17 @@ function checkDependencyAddedInWebpackConfig() {
 }
 
 function checkJSDependencyAddedInWebpackConfig() {
-  const jsconfig = WEBPACK_CONFIG.find((value) => {
-    return value.name === 'main-js-a-la-carte';
-  });
-  const nameCamel = camelCase(CLI_PACKAGE_JSON.name.replace('@material/', ''));
-  assert.notEqual(typeof jsconfig.entry[nameCamel], 'undefined',
-    'FAILURE: Component ' + CLI_PACKAGE_JSON.name + ' javascript dependency is not added to webpack ' +
-    'configuration. Please add ' + nameCamel + ' to ' + WEBPACK_CONFIG_RELATIVE_PATH + '\'s js-components ' +
-    'entry before commit.');
+  const name = getPkgName();
+  if (JS_WHITELIST.indexOf(name) === -1) {
```

Comment:
Let's restructure this to early exit via `return` if the name is included in the whitelist.

Reply:
Done